docs/index.rst

```diff
@@ -79,10 +79,3 @@ Main Qiskit-related projects
    GitHub <https://github.com/Qiskit/qiskit-terra>
    faq
 
-.. toctree::
-   :caption: Contributing
-   :hidden:
-
-   contributing_to_qiskit
-   deprecation_policy
-   maintainers_guide
```

Comment:
Did you mean to delete this file and contributing_to_qiskit.rst too?

Reply:
yeah jake asked me to remove it in the other contributing guide PR as well so just thought i'd do it in both to avoid merge conflicts whichever gets merged first

Reply:
<img width="779" alt="Screenshot 2023-11-17 at 16 40 27" src="https://github.com/Qiskit/qiskit/assets/23662430/00249ff1-84e8-474f-bbfa-51c7eda4a111">
although i realise now that plan has backfired 🫠

Reply:
Yeah, this PR will want to delete the maintainers guide and deprecation policy, but leave contributing guide. You'll get a merge conflict with your PR, but such it is 🤷